sandbox.js

```diff
@@ -0,0 +1,6 @@
+var character = 'Mario';

```

Comment:
For now this is fine, we try and stay away from `var` and only use `const`
https://dev.to/johnwolfe820/should-you-never-truly-use-var-bdi

But this is okay for this tutorial don't worry about it

Reply:
Aaaa I see this is within the .js file that you did not write, don't worry at all. My mistake